forch/forchestrator.py

```diff
@@ -81,13 +86,23 @@ def initialize(self):
         self._local_collector.initialize()
         self._cpn_collector.initialize()
         LOGGER.info('Using peer controller %s', self._get_peer_controller_url())
+
+        self._faucetizer = Faucetizer()
```

Comment:
this should be conditional somehow... e.g. based on something in the forch.yaml file, and if not enabled then _faucetizer should be None

Reply:
Done